codeHF/workflows.yml

```diff
@@ -28,6 +28,14 @@ workflows:
       default: [HFCANDP3BASE, HFCANDP3EXT]
       mc: [HFCANDP3MCREC, HFCANDP3MCGEN]
 
+  o2-analysis-hf-candidate-creator-xicc:
+    <<: *cand_creator
+    activate: no
+    dependencies: [o2-analysis-hf-xic-topkpi-candidate-selector]
```

Comment:
`activate` is obsolete.
Single dependency can be a simple string (instead of a list).